python/pyspark/sql/dataframe.py

```diff
@@ -6281,6 +6282,23 @@ def toPandas(self) -> "PandasDataFrameLike":
         """
         ...
 
+    @property
+    def executionInfo(self) -> Optional["ExecutionInfo"]:
+        """
+        Returns a QueryExecution object after the query was executed.
```

Comment:
Will it be overwritten if people execute the same DataFrame repeatedly?

Reply:
Yes, I will update the documentation.